dotCMS/src/main/java/com/dotmarketing/db/LocalTransaction.java

```diff
@@ -101,10 +101,16 @@ static public void wrap(final VoidDelegate delegate) throws DotDataException {
     } // wrap.
 
     private static void handleException(final boolean isLocalTransaction,
-                                        final Throwable  e) throws DotDataException {
+                                        final Throwable  e) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)